src/main/resources/wa-task-configuration-employment-et_englandwales_multiple.dmn

```diff
@@ -147,7 +147,8 @@
           <text></text>
         </inputEntry>
         <inputEntry id="UnaryTests_1sscd2z">
-          <text>"ReviewReferralJudiciaryMultiple"</text>
+          <text>"ReviewReferralJudiciaryMultiple",
+"ReviewReferralResponseJudiciaryMultiple"</text>
```

Comment:
Should there be an entry for Role Category, Description, the priorities, and dueDateIntervalDays as well?

Reply:
Added